third_party/terraform/resources/resource_bigquery_table.go

```diff
@@ -153,9 +153,9 @@ func resourceBigQueryTable() *schema.Resource {
 						"source_format": {
 							Type:        schema.TypeString,
 							Required:    true,
-							Description: `The data format. Supported values are: "CSV", "GOOGLE_SHEETS", "NEWLINE_DELIMITED_JSON", "AVRO", "PARQUET", and "DATASTORE_BACKUP". To use "GOOGLE_SHEETS" the scopes must include "googleapis.com/auth/drive.readonly".`,
+							Description: `The data format. Supported values are: "CSV", "GOOGLE_SHEETS", "NEWLINE_DELIMITED_JSON", "AVRO", "PARQUET", "DATSTORE_BACKUP", and "BIGTABLE". To use "GOOGLE_SHEETS" the scopes must include "googleapis.com/auth/drive.readonly".`,
```

Comment:
I wonder `ORC` can also included in the source_format list ?. We have an open issue (https://github.com/hashicorp/terraform-provider-google/issues/7691) for that type.

Reply:
Hi, sorry but that's out of scope of this PR - adding it in the list is straightforward but requires to write the tests.